frog/imports/api/remoteActivities.js

```diff
@@ -0,0 +1,36 @@
+import { uuid } from 'frog-utils';
+import { Activities } from '/imports/api/activities';
+
+export const removeActivity = (id: string) =>
+  fetch('http://icchilisrv4.epfl.ch:5000/activities?uuid=eq.'.concat(id), {
```

Comment:
you know you can also use 'string' + id ?

Reply:
can you make the http URL a separate const, something like

const RemoteServer = Meteor.settings.public.remoteServer || 'http://...'

and then reuse that in all the functions?